platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/sfp.py

```diff
@@ -5,35 +5,45 @@
     from sonic_platform_base.sonic_xcvr.sfp_optoe_base import SfpOptoeBase
     from sonic_platform.platform_thrift_client import thrift_try
     from sonic_platform.platform_thrift_client import pltfm_mgr_try
+    from sonic_platform.pltfm_mgr_rpc import ttypes
 except ImportError as e:
     raise ImportError (str(e) + "- required module not found")
 
 SFP_TYPE = "SFP"
 QSFP_TYPE = "QSFP"
 QSFP_DD_TYPE = "QSFP_DD"
+SFP_EEPROM_PATH = "/var/run/platform/sfp/"
+EEPROM_PAGE_SIZE = 128
 
+try:
+    from thrift.Thrift import TApplicationException
+
+    def cached_num_bytes_get(client):
```

Comment:
Can we move this as a part of `__init__()` ?

Reply:
It's better to keep it here, so it executes once and not every time constructor called